### ISSUE TITLE
No search results when using -i option

### ISSUE THREAD

**Issue by uzuto** [justchokingaround/jerry#1]

**Comment by justchokingaround** [comment]
could you please be more descriptive or send a screenshot?

**Comment by Pjarky** [comment]
The command seems to never give any search result. This happens when doing `jerry -i blue lock -n 5,` `jerry -i naruto -n 9`, `jerry -i Naruto -n 9` or `jerry -i "Naruto" -n 9`.
![image](https://user-images.githubusercontent.com/115619212/209568014-3ce61ee6-ab18-4a03-ae81-edcd0098ea76.png)
![image](https://user-images.githubusercontent.com/115619212/209568363-0db11177-dd7c-498e-9282-1527b9e8c718.png)
I am using Zorin/Ubuntu and used the exact commands in the README.MD file.

**Comment by justchokingaround** [comment]
fixed now, and also added an update flag. for now just update the script by uninstalling it:
```sh
sudo rm /usr/local/bin/jerry
```
and then reinstalling it again:
```sh
sudo curl -sL github.com/justchokingaround/jerry/raw/main/jerry.sh -o /usr/local/bin/jerry &&
sudo chmod +x /usr/local/bin/jerry
```
next time u will be able to just run
```sh
sudo jerry -u
```

**Comment by justchokingaround** [comment]
let me know if it fixed your issue

**Comment by uzuto** [comment]
> let me know if it fixed your issue

Thank you, It's working fine now and btw I like the fact that you use zoro for scraping where the whole world uses gogo.

https://user-images.githubusercontent.com/90568133/209766214-4d4051c6-d097-4c33-8822-e04e599c49d8.mp4

**Comment by justchokingaround** [comment]
any argument shenanigans not working should be fixed in 39aad98a0dc8f41687f184ccaa5a793443c085b1
